octobot_tentacles_manager/managers/tentacle_manager.py

```diff
@@ -36,7 +36,6 @@ async def install_tentacle(self, tentacle_path):
         tentacle_module_path = path.join(self.target_tentacle_path, self.tentacle.name)
         await self._update_tentacle_folder(tentacle_module_path)
         await managers.create_tentacle_init_file_if_necessary(tentacle_module_path, self.tentacle)
-        self.import_tentacle_config_if_any(tentacle_module_path)
```

Comment:
should we really remove this ? I think it's necessary to import the tentacle config into the user's tentacles reference configuration